eclipsePlugin/test/de/tobject/findbugs/reporter/ThrottledProgressMonitorTest.java

```diff
@@ -0,0 +1,52 @@
+package de.tobject.findbugs.reporter;
```

Comment:
Please do not add test code to the plugin. We have a dedicated tests plugin for tests.